docs/DEVELOPER-GUIDE.md

```diff
@@ -106,6 +106,22 @@ __NOTES__
 + The path are immutable: they must be chosen during creation (in the `__init__()` function), and must not be modified afterwards. The parent scanner (e.g.: `Zap`) should define the mount points, and each runtime (e.g.: `ZapPodman`) should fill each map *once*
 + For `type = None` (the scanner will run on the host), then the map must be the same (e.g.: `PathMap("/path/to/dir", "/path/to/dir")`)
 
+### Integrating the scanner to Defect Dojo
+
+The scanner can optionally be integrated to Defect Dojo. All it needs to do is to create an optional `data_for_defect_dojo()` method (no parameters).
```

Comment:
`The scanner can optionally be integrated to Defect Dojo. All it needs to do is to create an optional `data_for_defect_dojo()` method (no parameters).
This can be useful when the scanner exports a file that can be imported into a Defect Dojo test.` => I'd rephrase like "If a scanner is supported by Defect Dojo, the scanner can be configured to import its scan results to Defect Dojo automatically. All it needs to do is to create an optional `data_for_defect_dojo()` method (no parameters)"

WDYT?

Reply:
👍